ml-testing-toolkit/chart-backend/values.yaml

```diff
@@ -268,14 +268,9 @@ parameters: {}
 service:
   type: ClusterIP
   ports:
-    specApi:
-      name: "spec-api"
-      externalPort: 5000
-      internalPort: 5000
-    adminApi:
-      name: "admin-api"
-      externalPort: 5050
-      internalPort: 5050
+    name: "spec-api"
```

Comment:
oops, I didn't mean to touch this. I was using it as an example and I guess I lost track of the files.

Reply:
Ok, are you going to revert these?

Reply:
Thanks